### DISCUSSION TITLE
Does deepin customize the linux main kernel?

### DISCUSSION THREAD

**Discussion by Bottle-sys** [linuxdeepin/developer-center#7514] [Development Questions | 开发答疑] [answered]
Does deepin customize the linux main kernel? 
If yes,where I can download the deepin customized linux kernel source code?

**Comment by BLumia** [comment, marked as answer]
Yes, see https://github.com/deepin-community/kernel for the source code and feel free to join the `#deepin-kernel:deepin.org` matrix room if you are interested in contributing patches :)

**Reply by Bottle-sys** [reply]
Thank you！ I will try it.

**Reply by Bottle-sys** [reply]
Hi BLumia, One more question, I need to rebuild kernel code to clarify some HW issues. After installing Deep In version 20.9 which is formal released, I can see it's kernel verion is 15.18.17. Is this version of kernel code exactly  algined with github code tag 15.18.17（https://github.com/deepin-community/kernel/tree/v5.18.17/）？
Thank you!
